jib-core/src/test/java/com/google/cloud/tools/jib/builder/TestSourceFilesConfiguration.java

```diff
@@ -70,12 +81,51 @@ public String getClassesPathOnImage() {
     return EXTRACTION_PATH + "classes/";
   }
 
-  /** Lists the files in the {@code resourcePath} resources directory. */
-  private ImmutableList<Path> getFilesList(String resourcePath)
-      throws URISyntaxException, IOException {
-    try (Stream<Path> fileStream =
-        Files.list(Paths.get(Resources.getResource(resourcePath).toURI()))) {
-      return fileStream.collect(ImmutableList.toImmutableList());
+  public static Builder builder() {
```

Comment:
ordering nit: for convention in the codebase, we're trying to follow the class element ordering of:
- public->private
- static->non-static
- members->constructor->methods